[0m10:13:29.306994 [debug] [MainThread]: Got an exception trying to initialize tracking


============================== 10:13:29.324839 | a066b2a6-c0b8-4c19-8963-f3c9f0d27d6f ==============================
[0m10:13:29.324839 [info ] [MainThread]: Running with dbt=1.10.11
[0m10:13:29.324839 [debug] [MainThread]: running dbt with arguments {'no_print': 'None', 'quiet': 'False', 'partial_parse': 'True', 'profiles_dir': 'C:\\Users\\huytaq\\.dbt', 'invocation_command': 'dbt ', 'log_path': 'd:\\location-analytics-suggestions\\dbt_project\\logs', 'write_json': 'True', 'log_format': 'default', 'indirect_selection': 'eager', 'cache_selected_only': 'False', 'target_path': 'None', 'empty': 'None', 'printer_width': '80', 'use_colors': 'True', 'introspect': 'True', 'fail_fast': 'False', 'log_cache_events': 'False', 'send_anonymous_usage_stats': 'True', 'use_experimental_parser': 'False', 'static_parser': 'True', 'debug': 'False', 'version_check': 'True', 'warn_error': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])'}
[0m10:13:29.427446 [info ] [MainThread]: Warning: No packages were found in packages.yml
[0m10:13:29.428446 [info ] [MainThread]: Warning: No packages were found in packages.yml
[0m10:13:29.428446 [debug] [MainThread]: Command `cli deps` succeeded at 10:13:29.428446 after 0.16 seconds
[0m10:13:29.429445 [debug] [MainThread]: Flushing usage events
[0m10:23:01.639396 [debug] [MainThread]: Got an exception trying to initialize tracking


============================== 10:23:01.657354 | 67a83eb3-bc85-4228-800e-2795fb345750 ==============================
[0m10:23:01.657354 [info ] [MainThread]: Running with dbt=1.10.11
[0m10:23:01.658355 [debug] [MainThread]: running dbt with arguments {'warn_error': 'None', 'no_print': 'None', 'use_experimental_parser': 'False', 'profiles_dir': 'C:\\Users\\huytaq\\.dbt', 'invocation_command': 'dbt ', 'log_path': 'd:\\location-analytics-suggestions\\dbt_project\\logs', 'log_format': 'default', 'write_json': 'True', 'indirect_selection': 'eager', 'cache_selected_only': 'False', 'target_path': 'None', 'empty': 'None', 'printer_width': '80', 'use_colors': 'True', 'introspect': 'True', 'fail_fast': 'False', 'log_cache_events': 'False', 'send_anonymous_usage_stats': 'True', 'partial_parse': 'True', 'static_parser': 'True', 'debug': 'False', 'version_check': 'True', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])'}
[0m10:23:01.759096 [info ] [MainThread]: Warning: No packages were found in packages.yml
[0m10:23:01.760098 [info ] [MainThread]: Warning: No packages were found in packages.yml
[0m10:23:01.761097 [debug] [MainThread]: Command `cli deps` succeeded at 10:23:01.760098 after 0.16 seconds
[0m10:23:01.761097 [debug] [MainThread]: Flushing usage events
[0m11:07:25.111943 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000127C73020D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000127C9587190>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000127C9587070>]}


============================== 11:07:25.114941 | 5ed3b9c1-5596-4ce7-8c2c-0d10a0ea5398 ==============================
[0m11:07:25.114941 [info ] [MainThread]: Running with dbt=1.10.11
[0m11:07:25.114941 [debug] [MainThread]: running dbt with arguments {'warn_error': 'None', 'no_print': 'None', 'partial_parse': 'True', 'profiles_dir': './airflow/config', 'invocation_command': 'dbt debug --project-dir ./dbt_project --profiles-dir ./airflow/config', 'log_path': 'dbt_project\\logs', 'write_json': 'True', 'log_format': 'default', 'indirect_selection': 'eager', 'cache_selected_only': 'False', 'target_path': 'None', 'empty': 'None', 'printer_width': '80', 'use_colors': 'True', 'introspect': 'True', 'fail_fast': 'False', 'log_cache_events': 'False', 'send_anonymous_usage_stats': 'True', 'use_experimental_parser': 'False', 'static_parser': 'True', 'debug': 'False', 'version_check': 'True', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])'}
[0m11:07:25.124618 [info ] [MainThread]: dbt version: 1.10.11
[0m11:07:25.124618 [info ] [MainThread]: python version: 3.9.7
[0m11:07:25.124618 [info ] [MainThread]: python path: D:\location-analytics-suggestions\venv\Scripts\python.exe
[0m11:07:25.124618 [info ] [MainThread]: os info: Windows-10-10.0.26100-SP0
[0m11:07:25.171754 [info ] [MainThread]: Using profiles dir at ./airflow/config
[0m11:07:25.171754 [info ] [MainThread]: Using profiles.yml file at ./airflow/config\profiles.yml
[0m11:07:25.171754 [info ] [MainThread]: Using dbt_project.yml file at dbt_project\dbt_project.yml
[0m11:07:25.172754 [info ] [MainThread]: adapter type: clickhouse
[0m11:07:25.173757 [info ] [MainThread]: adapter version: 1.9.3
[0m11:07:25.245945 [info ] [MainThread]: Configuration:
[0m11:07:25.246944 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m11:07:25.246944 [info ] [MainThread]:   dbt_project.yml file [[32mOK found and valid[0m]
[0m11:07:25.246944 [info ] [MainThread]: Required dependencies:
[0m11:07:25.248017 [debug] [MainThread]: Executing "git --help"
[0m11:07:25.275459 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--no-lazy-fetch]\n           [--no-optional-locks] [--no-advice] [--bare] [--git-dir=<path>]\n           [--work-tree=<path>] [--namespace=<name>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone      Clone a repository into a new directory\n   init       Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add        Add file contents to the index\n   mv         Move or rename a file, a directory, or a symlink\n   restore    Restore working tree files\n   rm         Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect     Use binary search to find the commit that introduced a bug\n   diff       Show changes between commits, commit and working tree, etc\n   grep       Print lines matching a pattern\n   log        Show commit logs\n   show       Show various types of objects\n   status     Show the working tree status\n\ngrow, mark and tweak your common history\n   backfill   Download missing objects in a partial clone\n   branch     List, create, or delete branches\n   commit     Record changes to the repository\n   merge      Join two or more development histories together\n   rebase     Reapply commits on top of another base tip\n   reset      Reset current HEAD to the specified state\n   switch     Switch branches\n   tag        Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch      Download objects and refs from another repository\n   pull       Fetch from and integrate with another repository or a local branch\n   push       Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m11:07:25.275459 [debug] [MainThread]: STDERR: "b''"
[0m11:07:25.275459 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m11:07:25.276459 [info ] [MainThread]: Connection:
[0m11:07:25.276459 [info ] [MainThread]:   driver: None
[0m11:07:25.276459 [info ] [MainThread]:   host: clickhouse
[0m11:07:25.277459 [info ] [MainThread]:   port: 8123
[0m11:07:25.277459 [info ] [MainThread]:   user: default
[0m11:07:25.278459 [info ] [MainThread]:   schema: default
[0m11:07:25.278459 [info ] [MainThread]:   retries: 1
[0m11:07:25.278459 [info ] [MainThread]:   cluster: None
[0m11:07:25.279461 [info ] [MainThread]:   database_engine: None
[0m11:07:25.279461 [info ] [MainThread]:   cluster_mode: False
[0m11:07:25.279461 [info ] [MainThread]:   secure: False
[0m11:07:25.280458 [info ] [MainThread]:   verify: True
[0m11:07:25.280458 [info ] [MainThread]:   client_cert: None
[0m11:07:25.280458 [info ] [MainThread]:   client_cert_key: None
[0m11:07:25.280458 [info ] [MainThread]:   connect_timeout: 10
[0m11:07:25.280458 [info ] [MainThread]:   send_receive_timeout: 300
[0m11:07:25.280458 [info ] [MainThread]:   sync_request_timeout: 5
[0m11:07:25.281961 [info ] [MainThread]:   compress_block_size: 1048576
[0m11:07:25.282367 [info ] [MainThread]:   compression: 
[0m11:07:25.282367 [info ] [MainThread]:   check_exchange: True
[0m11:07:25.282367 [info ] [MainThread]:   custom_settings: None
[0m11:07:25.282965 [info ] [MainThread]:   use_lw_deletes: False
[0m11:07:25.282965 [info ] [MainThread]:   allow_automatic_deduplication: False
[0m11:07:25.282965 [info ] [MainThread]:   tcp_keepalive: False
[0m11:07:25.283965 [info ] [MainThread]: Registered adapter: clickhouse=1.9.3
[0m11:07:25.411193 [debug] [MainThread]: Acquiring new clickhouse connection 'debug'
[0m11:07:25.411193 [debug] [MainThread]: Opening a new connection, currently in state init
[0m11:07:37.263941 [debug] [MainThread]: dbt_clickhouse adapter: Got a retryable error when attempting to open a clickhouse connection.
1 attempts remaining. Retrying in 1 seconds.
Error:
Error HTTPConnectionPool(host='clickhouse', port=8123): Max retries exceeded with url: /? (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x00000127F19B09D0>: Failed to resolve 'clickhouse' ([Errno 11001] getaddrinfo failed)")) executing HTTP request attempt 1 (http://clickhouse:8123)
[0m11:07:46.362171 [info ] [MainThread]:   Connection test: [[31mERROR[0m]

[0m11:07:46.362171 [info ] [MainThread]: [31m1 check failed:[0m
[0m11:07:46.364092 [info ] [MainThread]: dbt was unable to connect to the specified database.
The database returned the following error:

  >Database Error
  Error HTTPConnectionPool(host='clickhouse', port=8123): Max retries exceeded with url: /? (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x00000127F19B0F10>: Failed to resolve 'clickhouse' ([Errno 11001] getaddrinfo failed)")) executing HTTP request attempt 1 (http://clickhouse:8123)

Check your database credentials and try again. For more information, visit:
https://docs.getdbt.com/docs/configure-your-profile


[0m11:07:46.365710 [debug] [MainThread]: Command `dbt debug` failed at 11:07:46.365710 after 21.32 seconds
[0m11:07:46.367702 [debug] [MainThread]: Connection 'debug' was left open.
[0m11:07:46.368709 [debug] [MainThread]: On debug: No close available on handle
[0m11:07:46.368709 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000127C73020D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000127C972C3D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000127C8705670>]}
[0m11:07:46.369706 [debug] [MainThread]: Flushing usage events
[0m11:07:47.555769 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m17:27:59.578847 [debug] [MainThread]: Got an exception trying to initialize tracking


============================== 17:27:59.596042 | d7c88a16-4fff-4ed0-9c08-dc7e52024ee7 ==============================
[0m17:27:59.596042 [info ] [MainThread]: Running with dbt=1.10.11
[0m17:27:59.596042 [debug] [MainThread]: running dbt with arguments {'quiet': 'False', 'no_print': 'None', 'use_experimental_parser': 'False', 'empty': 'None', 'log_path': 'd:\\location-analytics-suggestions\\dbt_project\\logs', 'invocation_command': 'dbt ', 'write_json': 'True', 'log_format': 'default', 'indirect_selection': 'eager', 'cache_selected_only': 'False', 'target_path': 'None', 'profiles_dir': 'C:\\Users\\huytaq\\.dbt', 'printer_width': '80', 'use_colors': 'True', 'introspect': 'True', 'fail_fast': 'False', 'send_anonymous_usage_stats': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'static_parser': 'True', 'debug': 'False', 'version_check': 'True', 'warn_error': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])'}
[0m17:27:59.743743 [info ] [MainThread]: Warning: No packages were found in packages.yml
[0m17:27:59.745301 [info ] [MainThread]: Warning: No packages were found in packages.yml
[0m17:27:59.746301 [debug] [MainThread]: Command `cli deps` succeeded at 17:27:59.746301 after 0.21 seconds
[0m17:27:59.746301 [debug] [MainThread]: Flushing usage events
[0m08:40:11.008032 [debug] [MainThread]: Got an exception trying to initialize tracking


============================== 08:40:11.038069 | 9bfa8590-8a1e-439d-9685-c75abed9ef80 ==============================
[0m08:40:11.038069 [info ] [MainThread]: Running with dbt=1.10.11
[0m08:40:11.039068 [debug] [MainThread]: running dbt with arguments {'empty': 'None', 'use_experimental_parser': 'False', 'no_print': 'None', 'warn_error': 'None', 'log_path': 'd:\\location-analytics-suggestions\\dbt_project\\logs', 'invocation_command': 'dbt ', 'log_format': 'default', 'write_json': 'True', 'indirect_selection': 'eager', 'cache_selected_only': 'False', 'target_path': 'None', 'profiles_dir': 'C:\\Users\\huytaq\\.dbt', 'printer_width': '80', 'use_colors': 'True', 'introspect': 'True', 'fail_fast': 'False', 'log_cache_events': 'False', 'send_anonymous_usage_stats': 'True', 'partial_parse': 'True', 'static_parser': 'True', 'debug': 'False', 'version_check': 'True', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])'}
[0m08:40:11.211566 [info ] [MainThread]: Warning: No packages were found in packages.yml
[0m08:40:12.747685 [info ] [MainThread]: Warning: No packages were found in packages.yml
[0m08:40:12.776523 [debug] [MainThread]: Command `cli deps` succeeded at 08:40:12.776523 after 1.82 seconds
[0m08:40:12.776523 [debug] [MainThread]: Flushing usage events
[0m09:02:45.800545 [debug] [MainThread]: Got an exception trying to initialize tracking


============================== 09:02:45.821772 | 3e8d80d9-8bf9-41eb-8183-6d450367e684 ==============================
[0m09:02:45.821772 [info ] [MainThread]: Running with dbt=1.10.11
[0m09:02:45.821772 [debug] [MainThread]: running dbt with arguments {'profiles_dir': 'C:\\Users\\huytaq\\.dbt', 'warn_error': 'None', 'no_print': 'None', 'use_experimental_parser': 'False', 'log_path': 'd:\\location-analytics-suggestions\\dbt_project\\logs', 'invocation_command': 'dbt ', 'log_format': 'default', 'write_json': 'True', 'indirect_selection': 'eager', 'cache_selected_only': 'False', 'target_path': 'None', 'empty': 'None', 'printer_width': '80', 'use_colors': 'True', 'introspect': 'True', 'fail_fast': 'False', 'log_cache_events': 'False', 'send_anonymous_usage_stats': 'True', 'partial_parse': 'True', 'static_parser': 'True', 'debug': 'False', 'version_check': 'True', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])'}
[0m09:02:45.946275 [info ] [MainThread]: Warning: No packages were found in packages.yml
[0m09:02:45.947275 [info ] [MainThread]: Warning: No packages were found in packages.yml
[0m09:02:45.949277 [debug] [MainThread]: Command `cli deps` succeeded at 09:02:45.948276 after 0.20 seconds
[0m09:02:45.949277 [debug] [MainThread]: Flushing usage events
[0m09:32:05.059819 [debug] [MainThread]: Got an exception trying to initialize tracking


============================== 09:32:05.079437 | ecea2909-8b7b-4ea0-9fcd-d823728240e7 ==============================
[0m09:32:05.079437 [info ] [MainThread]: Running with dbt=1.10.11
[0m09:32:05.080445 [debug] [MainThread]: running dbt with arguments {'use_experimental_parser': 'False', 'no_print': 'None', 'warn_error': 'None', 'profiles_dir': 'C:\\Users\\huytaq\\.dbt', 'log_path': 'd:\\location-analytics-suggestions\\dbt_project\\logs', 'invocation_command': 'dbt ', 'log_format': 'default', 'write_json': 'True', 'indirect_selection': 'eager', 'cache_selected_only': 'False', 'target_path': 'None', 'empty': 'None', 'printer_width': '80', 'use_colors': 'True', 'introspect': 'True', 'fail_fast': 'False', 'log_cache_events': 'False', 'send_anonymous_usage_stats': 'True', 'partial_parse': 'True', 'static_parser': 'True', 'debug': 'False', 'version_check': 'True', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])'}
[0m09:32:05.249670 [info ] [MainThread]: Warning: No packages were found in packages.yml
[0m09:32:05.250670 [info ] [MainThread]: Warning: No packages were found in packages.yml
[0m09:32:05.251672 [debug] [MainThread]: Command `cli deps` succeeded at 09:32:05.251672 after 0.25 seconds
[0m09:32:05.251672 [debug] [MainThread]: Flushing usage events
